### ISSUE TITLE
Random error

### ISSUE THREAD

**Issue by leesiongchan** [jimmynicol/image-resizer#91]
Not sure how to reproduce yet but here is the log, will further drill down and let you know the details.

```
/srv/image-resizer/node_modules/image-resizer/src/image.js:194
      this.format = imgType(data).ext;
                                 ^
TypeError: Cannot read property 'ext' of null
    at Image.Object.defineProperty.set (/srv/image-resizer/node_modules/image-resizer/src/image.js:194:34)
    at Request.<anonymous> (/srv/image-resizer/node_modules/image-resizer/src/streams/sources/external.js:65:28)
    at emitOne (events.js:82:20)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (/srv/image-resizer/node_modules/request/request.js:929:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at nextTickCallbackWith2Args (node.js:474:9)
    at process._tickDomainCallback (node.js:429:17)
```

**Comment by kvieira90** [comment]
Also receiving this error earlier today

**Comment by teohhanhui** [comment]
Since the maintainer is unresponsive, we've decided to maintain our own fork at https://github.com/tripviss/image-resizer

Fixed in tripviss/image-resizer#7

/cc @leesiongchan
